content/docs/authorization/v2.0 Tech Preview/configuration/proxy-server/_index.md

```diff
@@ -47,35 +47,59 @@ Afterwards, the storage administrator can configure Authorization with the follo
 
 ### Configuring Storage
 
-The storage types supported are `powerflex` and `powermax`. During the creation of a storage system and role, the storage type must be one of the supported types.
+The storage types supported are `powerflex`, `powermax`, and `powerscale`. During the creation of a storage system and role, the storage type must be one of the supported types.
 
-A `storage` entity in CSM Authorization consists of the storage type (`powerflex` or `powermax`), the system ID, the API endpoint, and the vault credentials path. For example, to create PowerFlex storage:
+A `storage` entity in CSM Authorization consists of the storage type (`powerflex`, `powermax` or `powerscale`), the system ID, the API endpoint, and the vault credentials path. Edit these parameters in the manifest:
+
+   | Parameter | Description | Required | Default |
+   | --------- | ----------- | -------- |-------- |
+   | type | The type of the stoage array. | Yes | - |
+   | endpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
+   | systemID | System ID of the backend storage array. | Yes | - |
+   | vault.identifier | The credential manager identifier to be used. | Yes | - |
+   | vault.kvEngine | The way that credentials for the storage array are stored. | Yes | secret |
```

Comment:
This is the path to the KV Engine that the user wants to use. 

"The path to the KV secrets engine"

Reply:
Fixed